### PR TITLE
feat: virtualize table rows

### PR DESCRIPTION
**Pull request by stropitek**

### CODE REVIEW

**Comment by stropitek** [comment]
There is a change in how borders are displayed when `striped` and `bordered` is combined.

I prefer the new one:

Before:
CleanShot 2024-12-02 at 16 55 57

After:
CleanShot 2024-12-02 at 16 55 25

**Comment by targos** [comment]
I don't see the difference, even from afar where I see both at the same time.

File: src/components/table/table_root.tsx [hunk 1]
```diff
@@ -1,38 +1,101 @@
...
+  /**
+   * Set to true to enable sticky header.
+   * When using sticky header with virtualized rows, the `<table>` element will be styled with `display: contents`.
+   * Any style applied through `tableProps.style` will no longer have any effect.
+   */
   stickyHeader?: boolean;
+  /**
+   * Use virtualized rows to optimize rendering.
+   * When virtualizing rows, and in order to prevent layout shifts,
+   * it is recommended that all cells have a fixed width.
```
Comment (targos) [inline src/components/table/table_root.tsx:63]:
```suggestion
   * it is recommended that all columns have a fixed width.
```

**Review by targos — CHANGES_REQUESTED** [review]
stickyHeader doesn't work after some scrolling with the Virtualized and Custom Tr Render stories.

**Comment by stropitek** [comment]
> I don't see the difference, even from afar where I see both at the same time.

The second one has "borders" (technically it's a `box-shadow`) in between rows.

**Review by targos — APPROVED** [review]

**Pull request merged**